server/app/views/admin/programs/ProgramImageView.java

```diff
@@ -172,6 +157,24 @@ private StorageUploadRequest createStorageUploadRequest(ProgramDefinition progra
     return publicStorageClient.getSignedUploadRequest(key, onSuccessRedirectUrl);
   }
 
+  /** Creates the <input type="file"> element needed for admins to upload program images. */
+  private DivTag createFileInputFormElement() {
```

Comment:
suggestion (non-blocking):  What would you think of putting this helper function in either `FileUploadViewStrategy.java` or `ViewUtils.java` so that we can reuse it if ever we have another file input or decide to switch the applicant file input to the USWDS component?   Maybe we could put "USWDS" in the name of the function to distinguish it from the old way of doing it?